modules/reindex/src/test/resources/rest-api-spec/test/delete_by_query/20_validation.yaml

```diff
@@ -6,6 +6,21 @@
         index: _all
 
 ---
+"no query logs deprecation":
+
+  - skip:
+      features: "warnings"
+      version: " - 5.2.99"
+      reason: warning headers were introduced in 5.0.1 and their format changed in 5.3.0
```

Comment:
I think the reason should be because a DBQ with an empty body wasn't deprecated until 5.4.0.